graphql/resolvers.go

```diff
@@ -2482,7 +2482,7 @@ func (r *taskResolver) FailedTestCount(ctx context.Context, obj *restModel.APITa
 	return failedTestCount, nil
 }
 
-// TODO: DEPRICATE
+// TODO: deprecated
```

Comment:
super nit: do we want the verb form deprecate since it's a todo?

Reply:
we probably do lol but i will remove this soon anyways so i think its alright